Dockerfile

```diff
@@ -39,6 +39,17 @@ RUN make install
 WORKDIR /usr/src/app/olm/python
 RUN make olm-python3
 
+WORKDIR /usr/src/app
+
+# installing SQLCipher
+RUN apt-get update && apt-get install -y \
+        sqlcipher \
+        libsqlcipher-dev \
+        libsqlcipher0 \
+        libssl-dev \
+        gcc
```

Comment:
Is installing gcc necessary? It it likely already used by make command above

Reply:
It's also an artifact of me trying to get sql cipher to work. It can be deleted.

Reply:
I did some tests and it can't be deleted. Debian gets confused.